site/content/en/docs/reference/kubectl-kueue.md

```diff
@@ -75,14 +75,22 @@ You can run `kubectl kueue help` in the terminal to get the full list of command
 
 The following table includes short descriptions and the general syntax for all of the kueuectl operations:
 
-| Operation | Syntax                       | Description                                             |
-|--------|------------------------------|---------------------------------------------------------|
-| create | kubectl kueue create [TYPE] [NAME] [flags] | Create resource. | 
+| Operation | Syntax                                     | Description          |
+|-----------|--------------------------------------------|----------------------|
+| create    | kubectl kueue create [TYPE] [NAME] [flags] | Create a resource.   | 
```

Comment:
Will there be a more detailed documentation on the command (like what types are supported, what are the flags etc)?

Reply:
You mean to create similar table for each operation?

Reply:
Added

Reply:
How will a user know that create workload is not supported?

Reply:
Good point

Reply:
Added. But I think to split it by pages. WDYT?

Reply:
Done